src/simplify_qdq.cpp

```diff
@@ -49,6 +49,90 @@ std::unordered_set<std::string> get_quantizable_op_names()
     return s;
 }
 
+/// Recursively skip input instructions with one input that have a name in the skip_set.
+instruction_ref skip_unary_ins_from(instruction_ref ins, std::unordered_set<std::string> skip_set)
```

Comment:
This can already be done with `match::skip(match::name(...))`.

Reply:
For removing the q/dq pairs, not going to refactor for now.